libbeat/esclientleg/api_integration_test.go

```diff
@@ -164,3 +172,65 @@ func TestIngest(t *testing.T) {
 
 	assert.Equal(t, "test", doc.Field)
 }
+
+// FIXME: Below code is repeated in outputs/elasticsearch/internal/testing.go
```

Comment:
It would be nice to have some test utilities, somewhat similar to `net/http/httptest` package. Then we can share common test functionality. How about naming it `libbeat/esclientleg/eslegtest`? Alternatively we can also rename this package introduce the package `libbeat/esleg/eslegclient` and `libbeat/esleg/eslegtest`. WDYT?

Reply:
I created `libbeat/esclientleg/eslegtest` and moved the common test utility code into that package. See 27ee0f9.